proposals/dao/tip_vebalotc.ts

```diff
@@ -0,0 +1,183 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  PcvStats,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { BigNumber } from 'ethers';
+import { getImpersonatedSigner, overwriteChainlinkAggregator, time } from '@test/helpers';
+import { TransactionResponse } from '@ethersproject/providers';
+import { forceEth } from '@test/integration/setup/utils';
+
+const PROXY_ADMIN_STORAGE_SLOT = '0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103';
+
+const fipNumber = 'vebalotc';
+
+const vebalOtcBuyer = '0x6ef71cA9cD708883E129559F5edBFb9d9D5C6148';
+const vebalOtcAmount = ethers.utils.parseEther('1000000'); // 1M
```

Comment:
Personal preference but prefer `ethers.constants.WeiPerEther.mul(1_000_00)` syntax for these so can easily see order of magnitude

Reply:
ok